rllib/models/utils.py

```diff
@@ -153,6 +153,69 @@ def get_activation_fn(
     )
 
 
+@DeveloperAPI
+def get_initializer_fn(name: Optional[Union[str, Callable]], framework: str = "torch"):
+    # Already a callable return as is.
+    if callable(name):
+        return name
+
+    if framework == "torch":
+        name_lower = name.lower() if isinstance(name, str) else name
+        # If `None` we use the default initializer.
```

Comment:
Do this (together with "tf") as an early-out above. Similar to how you already handle callables.

Reply:
Done